api/query/concrete_query.go

```diff
@@ -117,6 +117,15 @@ func (Link) Size() int { return 1 }
 // Size of Count is the sum of the sizes of its constituent ConcretQuery instances.
 func (c Count) Size() int { return size(c.Args) }
 
+// Size of Is depends on the quality.
+func (q MetadataQuality) Size() int {
+	switch q {
```

Comment:
`return 1` with a comment and/or assert would be better IMHO than scaffolding for extensions that may never come.